frontstage/controllers/conversation_controller.py

```diff
@@ -30,7 +31,7 @@ def get_conversation(thread_id):
     url = '{}v2/threads/{}'.format(current_app.config["RAS_SECURE_MESSAGING_SERVICE"], thread_id)
 
     try:
-        response = session.get(url, headers=headers)
+        response = get_session().get(url, headers=headers)
```

Comment:
The documentation suggests Session() should be used as a context manager to ensure the connection is closed after the request i.e. `with get_session() as s:` http://docs.python-requests.org/en/master/user/advanced/

Reply:
I expect we could also decorate the session-based functions with `@get_session`, (after tweaking it a bit) which would then inject the session variable as a parameter to be used throughout.

Reply:
Fair point @jcox-dev. I find it's a little harder to follow when there is a decorator that injects variables